Client/src/Unified/Services/Submitter/BatchUntilInactiveBlock.cs

```diff
@@ -0,0 +1,249 @@
+  {
+    var offerResult = ((ITargetBlock<T>)source_).OfferMessage(messageHeader,
+                                                              messageValue,
+                                                              source,
+                                                              consumeToAccept);
+
+    if (offerResult == DataflowMessageStatus.Accepted)
+    {
+      timer_.Change(Timeout,
+                    System.Threading.Timeout.InfiniteTimeSpan);
+    }
+
+    return offerResult;
+  }
+
+  /// <summary>
+  ///   Called by a linked ITargetBlock TInput  to accept and consume a
+  ///   DataFlowMessageHeader previously offered by this ISourceBlockTOutput .
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="target"></param>
+  /// <param name="messageConsumed"></param>
+  /// <returns></returns>
+  public T[]? ConsumeMessage(DataflowMessageHeader messageHeader,
+                             ITargetBlock<T[]?>    target,
+                             out bool              messageConsumed)
+    => ((ISourceBlock<T[]?>)source_).ConsumeMessage(messageHeader,
+                                                    target,
+                                                    out messageConsumed);
+
+  /// <summary>
+  ///   Called by a linked ITargetBlock TInput  to reserve a previously offered
+  ///   DataFlowMessageHeader by this ISourceBlockTOutput .
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="target"></param>
+  /// <returns></returns>
+  public bool ReserveMessage(DataflowMessageHeader messageHeader,
+                             ITargetBlock<T[]>     target)
+    => ((ISourceBlock<T[]>)source_).ReserveMessage(messageHeader,
+                                                   target);
+
+  /// <summary>
+  ///   Called by a linked ITargetBlock  TInput  to release a previously reserved DataflowMessageHeader by this
+  ///   ISourceBlock;.
+  /// </summary>
+  /// <param name="messageHeader"></param>
+  /// <param name="target"></param>
+  public void ReleaseReservation(DataflowMessageHeader messageHeader,
+                                 ITargetBlock<T[]>     target)
+    => ((ISourceBlock<T[]>)source_).ReleaseReservation(messageHeader,
+                                                       target);
+
+  /// <summary>
+  ///   Attempts to synchronously receive an available output item from the IReceivableSourceBlock TOutput
+  /// </summary>
+  /// <param name="filter"></param>
+  /// <param name="item"></param>
+  /// <returns></returns>
+  public bool TryReceive(Predicate<T[]>? filter,
+                         out T[]         item)
+    => source_.TryReceive(filter,
+                          out item);
+
+  /// <summary>
+  ///   Attempts to synchronously receive all available items from the IReceivableSourceBlock TOutput
+  /// </summary>
+  /// <param name="items"></param>
+  /// <returns></returns>
```

Comment:
You need a description for all the params
Maybe you can use directly the following:
```c#
/// <inherit_doc/>
```